src/routes/languages.js

```
@@ -0,0 +1,16 @@
+const asyncWrapper = require('~/middlewares/asyncWrapper')
+const Language = require('~/models/languages')
+const isEntityValid = require('~/middlewares/entityValidation')
+
+const languageController = require('~/controllers/languages')
+
+const params = [{ model: Language, idName: 'id' }]
+
+router.get('/', asyncWrapper(languageController.getLanguages))
+router.get('/:id', isEntityValid({ params }), asyncWrapper(languageController.getLanguageById))
+router.post('/', asyncWrapper(languageController.createLanguage))
+router.patch('/:id', isEntityValid({ params }), asyncWrapper(languageController.updateLanguage))
```

ESLint

[error] Expected linebreaks to be 'CRLF' but found 'LF'.

(linebreak-style)